demo/src/SvgDisplay.tsx

```diff
@@ -1,16 +1,21 @@
 import React, { useRef, useEffect } from "react";
 import mem from "../../src/index"; // TODO: replace with import of the package after it's been published
+import { Typography } from "@mui/material";
 
 type SvgDisplayPropTypes = {
     jsonResult: object;
+    setSvgResult: React.Dispatch<React.SetStateAction<string>>;
 };
 
 export default function SvgDisplay(props: SvgDisplayPropTypes) {
     const canvasRef = useRef(null);
+    const canvasWidth = 1300;
+    const canvasHeight = 1000;
 
     useEffect(() => {
         if (props.jsonResult !== null) {
-            const m = mem.draw(props.jsonResult, true, { width: 1300 });
+            const m = mem.draw(props.jsonResult, true, { width: canvasWidth });
```

Comment:
I realized when testing the download that `mem.draw` is actually mutating the input JSON.

This is something we can change later, but for now you should make a deepcopy of `props.jsonResult` to pass in

Reply:
Node 17 added structuredClone, the built-in deep clone method and I'm using that - unfortunately the jsdom environment has had trouble loading that for a few years. Using the workaround from https://togithub.com/jsdom/jsdom/issues/3363#issuecomment-1467894943 for now.